Stripe/STPImageLibrary.m

```diff
@@ -112,26 +116,33 @@ + (UIImage *)brandImageForCardBrand:(STPCardBrand)brand
     BOOL shouldUseTemplate = isTemplate;
     NSString *imageName;
     switch (brand) {
-            case STPCardBrandAmex:
+        case STPCardBrandAmex:
             imageName = shouldUseTemplate ? @"stp_card_amex_template" : @"stp_card_amex";
             break;
-            case STPCardBrandDinersClub:
+        case STPCardBrandDinersClub:
             imageName = shouldUseTemplate ? @"stp_card_diners_template" : @"stp_card_diners";
             break;
-            case STPCardBrandDiscover:
+        case STPCardBrandDiscover:
             imageName = shouldUseTemplate ? @"stp_card_discover_template" : @"stp_card_discover";
             break;
-            case STPCardBrandJCB:
+        case STPCardBrandJCB:
             imageName = shouldUseTemplate ? @"stp_card_jcb_template" : @"stp_card_jcb";
             break;
-            case STPCardBrandMasterCard:
+        case STPCardBrandMasterCard:
             imageName = shouldUseTemplate ? @"stp_card_mastercard_template" : @"stp_card_mastercard";
             break;
-            case STPCardBrandUnknown:
+        case STPCardBrandUnionPay:
+            if ([[[NSLocale currentLocale] localeIdentifier].lowercaseString hasPrefix:@"zh"]) {
```

Comment:
🇨🇳

Reply:
I think my last change was changing this call to `localeIdentifier` instead of `languageCode` because the latter is available iOS 10+